src/stylesheets/components/_resultsList.scss

```diff
@@ -74,32 +79,30 @@
 }
 
 .action-menu-wrapper {
-  height:100%;
+  height: 100%;
   display: flex;
   align-items: flex-end;
   justify-content: center;
   flex-direction: column;
 }
 
 .action-menu-placeholder {
-  width: 38px;
-  height: 38px;
+  width: 30px;
+  height: 30px;
   border-radius: 50%;
   border: solid 1px #ccc;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  color:#666;
-  font-size:1.5em;
+  color: #666;
+  font-size: 1em;
   cursor: pointer;
+  margin-top: 4px;
 }
 
-h2 .action-menu-placeholder {
-  font-size:1em;
-}
-
-div[class^="grid-col-"] .sds-tag--static, div[class*=" grid-col-"] .sds-tag--static {
+div[class^="grid-col-"] .sds-tag--static,
```

Comment:
This will affect all the tags inside a grid-col not only the tags located in the searchlist component

Reply:
Nothing else was using this class, but I renamed and moved this to tags and created a block style modifier rather than calling it static.